### PR TITLE
Feature/kubeflow ci

### PR DESCRIPTION
**Pull request by ycliuhw**
*add new CI test for kubeflow: assess_caas_deploy_kubeflow.py*

## Checklist

 - [ ] ~Requires a [pylibjuju](https://github.com/juju/python-libjuju) change~
 - [ ] ~Added [integration tests](https://github.com/juju/juju/tree/develop/tests) for the PR~
 - [ ] ~Added or updated [doc.go](https://discourse.jujucharms.com/t/readme-in-packages/451) related to packages changed~
 - [x] Comments answer the question of why design decisions were made

## QA steps

```console
$ ./acceptancetests/assess_caas_deploy_kubeflow.py \
    gke $GOPATH/bin/juju /tmp/artifacts/ nw-deploy-kubeflow-gke --caas-image-repo=ycliuhw \
    --caas-provider=GKE|MICROK8S|AKS|EKS \
    --k8s-controller \
    --bundle full|edge|lite [--build]

```

### CODE REVIEW

**Comment by ycliuhw** [comment]
[setup Kubeflow CI test](https://trello.com/c/oIXLsWDD/1390-setup-kubeflow-ci-test)

File: acceptancetests/requirements.txt [hunk 1]
```diff
@@ -1,28 +1,31 @@
 apache-libcloud>=2.4.0
-azure-batch>=0.30.0rc3
```
Comment (ycliuhw) [inline acceptancetests/requirements.txt:2]: These dependencies are really ancient

**Review by wallyworld — APPROVED** [review]
Nice. Thank you.

**Comment by ycliuhw** [comment]
$$merge$$

**Pull request merged**